<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="loadWeb">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>LOAD WEB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listView">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="mainPage">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="controlLayout" stretch="2">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QPushButton" name="startServer">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>46</height>
               </size>
              </property>
              <property name="text">
               <string>START SERVER</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextBrowser" name="outputBrowser">
            <property name="minimumSize">
             <size>
              <width>660</width>
              <height>300</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;OUTPUT:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;-------------------------------------------------------&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="serverConfigPage">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTabWidget" name="serverConfigGroup">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="serverConfigUI">
             <property name="minimumSize">
              <size>
               <width>640</width>
               <height>200</height>
              </size>
             </property>
             <attribute name="title">
              <string>Tab 1</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
              <item>
               <layout class="QVBoxLayout" name="serverConfigListView" stretch="1,0,1,0,1,0">
                <property name="sizeConstraint">
                 <enum>QLayout::SetDefaultConstraint</enum>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="localLayout" stretch="0,0">
                  <property name="topMargin">
                   <number>10</number>
                  </property>
                  <property name="bottomMargin">
                   <number>10</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="localView">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="layoutDirection">
                     <enum>Qt::LeftToRight</enum>
                    </property>
                    <property name="text">
                     <string>LOCAL: </string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                    <property name="wordWrap">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="local">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>30</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="portLayout">
                  <property name="topMargin">
                   <number>10</number>
                  </property>
                  <property name="bottomMargin">
                   <number>10</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="portView">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="text">
                     <string>PORT: </string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="port">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>30</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="configPathLayout">
                  <property name="topMargin">
                   <number>10</number>
                  </property>
                  <property name="bottomMargin">
                   <number>10</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="configPathView">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>CONFIGPATH: </string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="configPath">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>30</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="homeLayout">
                  <property name="topMargin">
                   <number>10</number>
                  </property>
                  <property name="bottomMargin">
                   <number>10</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="homeView">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>HOME: </string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="home">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>30</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="debugLayout">
                  <property name="topMargin">
                   <number>10</number>
                  </property>
                  <property name="bottomMargin">
                   <number>10</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="debugView">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>DEBUG: </string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="debug">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>30</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPushButton" name="saveServerConfigUiData">
                  <property name="text">
                   <string>SAVE</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="serverConfigJSON">
             <property name="minimumSize">
              <size>
               <width>640</width>
               <height>0</height>
              </size>
             </property>
             <attribute name="title">
              <string>Tab 2</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="serverConfigLayout">
                <item>
                 <widget class="QLabel" name="serverConfigView">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>SERVER_CONFIG.json</string>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="saveServerConfigJsonData">
                  <property name="text">
                   <string>SAVE</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTextEdit" name="serverConfigJSONData"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="rulesConfigPage">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QTabWidget" name="ruleConfigGroup">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="ruleConfigJSON">
             <attribute name="title">
              <string>Tab 2</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <layout class="QHBoxLayout" name="ruleConfigLayout">
                <item>
                 <widget class="QLabel" name="ruleConfigView">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>RULE_CONFIG.json</string>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="saveRuleConfigJsonData">
                  <property name="text">
                   <string>SAVE</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTextEdit" name="ruleConfigJSONData"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>856</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>MENU</string>
    </property>
    <addaction name="actionSTART_SERVER"/>
    <addaction name="actionOPEN_CONFIG"/>
    <addaction name="actionOPEN_RECENTLY_WEB"/>
    <addaction name="actionOPEN_WEB"/>
   </widget>
   <widget class="QMenu" name="menuSETTING">
    <property name="title">
     <string>SETTING</string>
    </property>
    <addaction name="actionDARK_MODE"/>
    <addaction name="actionOPEN_FOLDER"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuSETTING"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSTART_SERVER">
   <property name="text">
    <string>START SERVER</string>
   </property>
  </action>
  <action name="actionOPEN_CONFIG">
   <property name="text">
    <string>OPEN CONFIG</string>
   </property>
  </action>
  <action name="actionOPEN_WEB_VIEW">
   <property name="text">
    <string>OPEN WEB VIEW</string>
   </property>
  </action>
  <action name="actionOPEN_RECENTLY_WEB">
   <property name="text">
    <string>OPEN RECENTLY WEB</string>
   </property>
  </action>
  <action name="actionOPEN_WEB">
   <property name="text">
    <string>OPEN WEB</string>
   </property>
  </action>
  <action name="actionDARK_MODE">
   <property name="text">
    <string>DARK MODE</string>
   </property>
  </action>
  <action name="actionOPEN_FOLDER">
   <property name="text">
    <string>OPEN FOLDER</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
